workflow/log-monitoring/log-explorer.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Log Explorer
+sidebarTitle: Log Explorer
+---
+
+The Log Explorer is where all of your historical log data is located, including the date and time the log was fired as well as its corresponding resource attributes. This section is useful for accessing raw log resource attributes, drilling into a single host's metrics and processes, and tracing requests across your distributed microservices.
```

Comment:
The second sentence is much better as a "why would you care". Just a minor edit:

"This section is useful for understanding logs at a deeper level. By drilling into a log you'll be able to see raw log resource attributes, the associated host's metrics and processes, and any traces related to the log."